pkg/plugin/v2/scaffolds/internal/templates/controller/controller.go

```diff
@@ -29,6 +29,11 @@ var _ file.Template = &Controller{}
 type Controller struct {
 	file.TemplateMixin
 	file.MultiGroupMixin
+
+	// WireResource defines the api resources are generated or not.
+	// Ordered the struct this way to save the memory by organizing struct
```

Comment:
These template structs will be one-of per execution at most, and discarded pretty fast so memory efficiency is the last thing to worry about. Place it after the 4 mixins and use a `//nolint:maligned` to avoid the linter error.